spec/javascript/components/applications/__snapshots__/ApplicationEditPage.test.js.snap

```diff
@@ -1614,7 +1614,6 @@ Array [
                                 onChange={[Function]}
                                 onFocus={[Function]}
                                 type="checkbox"
-                                value={true}
```

Comment:
These values are removed from the snapshots due to default values being assumed by the Final Form fields